quarkus-extension/simple-rest-example/src/main/resources/process.bpmn

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_136l9dc" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.7.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.14.0">
+  <bpmn:process id="quarkus-process" isExecutable="true">
+    <bpmn:startEvent id="StartEvent_1">
+      <bpmn:outgoing>Flow_0kby13t</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:sequenceFlow id="Flow_0kby13t" sourceRef="StartEvent_1" targetRef="Activity_1eiwqxl" />
+    <bpmn:endEvent id="Event_155wmgi">
+      <bpmn:incoming>Flow_1a4ohec</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1a4ohec" sourceRef="Activity_1eiwqxl" targetRef="Event_155wmgi" />
+    <bpmn:serviceTask id="Activity_1eiwqxl" name="Call &#34;myServiceDelegate&#34;"
+                      camunda:delegateExpression="${myServiceDelegate}">
```

Comment:
```suggestion
                      camunda:delegateExpression="${myServiceDelegate}"
                      camunda:asyncBefore="true">
```

@koevskinikola, let's apply this change when camunda/camunda-bpm-platform#1542 is merged.